.circleci/config.yml

```diff
@@ -79,6 +84,29 @@ jobs:
           key: >-
             gradle-{{ checksum "/tmp/gradle_cache_seed" }}-{{ checksum
             ".circleci/config.yml" }}-macos
+  build-windows:
+    executor:
+        name: win/default
+        size: large
```

Comment:
This was the biggest executor available on my freebie CircleCI account. The project was using bigger executors than were available, so I adjusted them down to large on my working branch.

You may want to consider raising this for the official project.